src/routes/file_info_index/file_info_index_blueprint.py

```diff
@@ -91,7 +90,8 @@ def reindex_dataset_files_docs():
         fworker = FileWorker(appConfig=app_config, requestHeaders=request.headers)
 
         # Verify the user is a Data Admin, who can index all datasets
-        fworker.verify_op_permission(aDataset=None)
+        if not fworker.verify_data_admin():
```

Comment:
Essentially we don't need this Data-Admin group check here because the AWS API Gateway handles the auth check before allowing access to this `/datasets/reindex-all` endpoint. It'll be a duplicate check here in the code and it'll always pass.

Reply:
I understood the code would duplicate functionality you add in AWS Gateway, after one of our conversations.  But it was not clear to me how to verify functionality during development.  Does the group recommend coding & testing data admin functionality, then stripping it out?  Or just code knowing it will eventually work?  I understand my code inside the `if` clause will never be reached, thanks to AWS Gateway.

Reply:
Got it. We can just leave the code there, no harm. Might add a line of comment mentioning the API Gateway does the check?

Reply:
I put in a couple of comments.  It makes an interesting discussion: fully trust configuration to restrict the endpoint, but not so much to keep the slash off the end of URLs? ;) But I put one comment adjacent to the routes() decorator, and maybe we want to standardize a way to coordinate endpoint expectations in code with AWS Gateway configuration tasks?